applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -77,6 +62,59 @@ Make sure that the DFU lock utility is enabled if your nRF Desktop application c
 You cannot use this module with the :ref:`caf_ble_smp`.
 In other words, you cannot simultaneously enable the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option and the :kconfig:option:`CONFIG_CAF_BLE_SMP` Kconfig option.
 
+You have to choose backend for the DFU process by enabling one of the following options:
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT` - This option depends on the MCUboot bootloader support for the DFU process.
+    For details, see :ref:`mcuboot` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_SUIT` - This option depends on the SUIT support for the DFU process.
+    For details, see :ref:`ug_nrf54h20_suit_dfu` in the Zephyr documentation.
+
+.. note::
+    Currently b0 backend is not supported.
+
+.. _nrf_desktop_dfu_mcumgr_mcuboot:
+
+MCUmgr configuration using MCUboot backend
+==========================================
+
+The MCUboot backend configuration requires the following options:
+
+* MCUmgr groups:
+
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_IMG` - This option enables the MCUmgr image management handlers, which are required for the DFU process.
+    For details, see :ref:`zephyr:device_mgmt` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_MUTEX`
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_OS` - This option enables the MCUmgr OS management handlers, which are required for the DFU process.
+    For details, see :ref:`zephyr:device_mgmt` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_IMG_MANAGER`
+  * :kconfig:option:`CONFIG_STREAM_FLASH`
+  * :kconfig:option:`CONFIG_MCUBOOT_BOOTUTIL_LIB`
+
+To use the module with this backend, you must also enable the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option.
+
+By default, the MCUmgr DFU module confirms the image using the :c:func:`boot_write_img_confirmed` function during the system boot.
+If the :kconfig:option:`CONFIG_DESKTOP_DFU_MCUMGR_MCUBOOT_DIRECT_XIP` option is enabled, the MCUmgr DFU module assumes that the bootloader simply boots the image with a higher version and does not confirm the newly updated image after a successful boot.
+Make sure that :kconfig:option:`CONFIG_DESKTOP_DFU_MCUMGR_MCUBOOT_DIRECT_XIP` Kconfig option is enabled if devices use the MCUboot bootloader in direct-xip mode.
```

Comment:
`direct-xip mode without revert.` (since revert support was added, I would suggest to be more precise here)